apstools/beamtime/bss_info.py

```diff
@@ -0,0 +1,470 @@
+#!/usr/bin/env python
+
+"""
+Retrieve specific records from the APS Proposal and ESAF databases
+
+BSS: Beamline Scheduling System
+
+EXAMPLES::
+
+    bss_info current
+    bss_info esaf 226319
+    bss_info proposal 66083 2020-2 9-ID-B,C
+"""
+
+import argparse
+import datetime
+import dm               # APS data management library
+from bss_info_ophyd import EpicsBssDevice
+import os
+import pyRestTable
+import sys
+import time
+import yaml
+
+DM_APS_DB_WEB_SERVICE_URL = "https://xraydtn01.xray.aps.anl.gov:11236"
+CONNECT_TIMEOUT = 5
+POLL_INTERVAL = 0.01
+
+api_bss = dm.BssApsDbApi(DM_APS_DB_WEB_SERVICE_URL)
+api_esaf = dm.EsafApsDbApi(DM_APS_DB_WEB_SERVICE_URL)
+
+_cache_ = {}
+
+
+class EpicsNotConnected(Exception): ...
```

Comment:
Codacy found an issue: [Statement seems to have no effect](https://app.codacy.com/gh/BCDA-APS/apstools/pullRequest?prid=5764884)